build/config/phone/custom-prefs.js

```diff
@@ -2,3 +2,9 @@
 user_pref('devtools.responsiveUI.customWidth', 320);
 user_pref('devtools.responsiveUI.customHeight', 480);
 user_pref('devtools.responsiveUI.currentPreset', 'custom');
+
+user_pref('dom.caches.enabled', true);
+user_pref('dom.serviceWorkers.enabled', true);
+user_pref('dom.webcomponents.enabled', true);
+user_pref('dom.apps.developer_mode', true);
+user_pref('network.disable.ipc.security', true);
```

Comment:
Could we make sure that we don't ship a build with these prefs on? (i.e. only add them for debug builds)
This kind of things tend to be forgotten.